main.py

```diff
@@ -1,139 +1,122 @@
-        if time_diff > timedelta(days=stale_days):
+        if time_diff > timedelta(days=msg.get("stale_days")):
             print(f"Pull request: {pr.number} is stale!")
-            pr.create_issue_comment(msg_job2)
+            pr.create_issue_comment( msg.get("stale_label") )
             pr.add_to_labels('Stale')
-    except Exception as e:
-        print(f"Error occurred while processing pull request: {pr.number}")
-        print(f"Error: {str(e)}")
-
-# 3.close staled PR if 2 days of no activity
-for pr in pulls:
-    # check if the Stale label is applied on PR
-    if "Stale" in [label.name for label in pr.labels]:
-        try:
-            time_diff = now - pr.updated_at
-            # check if the time difference is greater than the stale_close_days
-            if time_diff > timedelta(days=stale_close_days):
-                print(f"Pull request: {pr.number} is stale and closed!")
-                pr.edit(state="closed")
-                pr.create_issue_comment(msg_job3)
-                print(msg_job3)
-        except Exception as e:
-            print(f"Error occurred while closing pull request: {pr.number} ")
-            print(f"Error: {str(e)}")
-
-print(f"pr_updated_at: {pr.updated_at}")
-
-# 4.Check if the pull request targets the master branch directly
-for pull in pulls:
-    if pull.base.ref == 'master' and not pull.head.ref.startswith('release/'):
-        try:
+
+    # 2.close staled PR if 2 days of no activity
+    if pulls.totalCount != 0:
+        for pr in pulls:
+            # check if the Stale label is applied on PR
+            if "Stale" in [label.name for label in pr.labels]:
+                time_diff = now - pr.updated_at
+                # check if the time difference is greater than the stale_close_days
+                if time_diff > timedelta(days=msg.get("stale_close_days")):
+                    print(f"Pull request: {pr.number} is stale and closed!")
+                    pr.edit(state="closed")
+                    pr.create_issue_comment(msg.get("staled_PR_closing") )
+                    print(msg.get("staled_PR_closing"))
+        print(f"pr_updated_at: {pr.updated_at}")
+
+    # 3.Check if the pull request targets the master branch directly
+    for pull in pulls:
+        if pull.base.ref == 'master' and not pull.head.ref.startswith('release/'):
             print(f"Pull request: {pull.number} was targeted to master")
             pull.edit(state='closed')
-            pull.create_issue_comment(msg_job4)
-            print(msg_job4)
-        except Exception as e:
-            print(f"Error occurred while processing pull request: {pull.number} ")
-            print(f"Error:  {e} ")
-
-# 5.Check if the pull request has a description
-for pull in pulls:
-    if not pull.body:
-        try:
+            pull.create_issue_comment(msg.get("check_PR_target") )
+            print(msg.get("check_PR_target"))
+
+    # 4.Check if the pull request has a description
+    for pull in pulls:
```

Comment:
@Arun9015  We do not need another iteration to be initiated. We should manage the condition in the previous iteration